src/lib/annotations/Annotator.js

```diff
@@ -705,10 +694,7 @@ class Annotator extends EventEmitter {
      * @return {void}
      */
     unbindCustomListenersOnThread(thread) {
-        thread.removeAllListeners('threaddeleted');
-        thread.removeAllListeners('threadcleanup');
-        thread.removeAllListeners('annotationsaved');
-        thread.removeAllListeners('annotationevent');
+        thread.removeAllListeners('threadevent');
```

Comment:
I think if you remove the args this should work. Make sure they're actually getting removed though